### ISSUE TITLE
should the commands the user enters in be imperative or infinitive?

### ISSUE THREAD

**Issue by jtauber** [jtauber/labe-luxnos#1]
I asked on twitter what other languages use for the verbs in “go east” or “open door” or “take lamp” or "read note".

**Comment by jtauber** [comment]
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">French uses mainly infinitive. Like recipes. Possible you could also find (but I did not check) indicative present if the game is verbose. With vouvoiement : &quot;vous prenez la fuite.&quot;</p>&mdash; Thibault Clerice (@PonteIneptique) <a href="https://twitter.com/PonteIneptique/status/979607340205989888?ref_src=twsrc%5Etfw">March 30, 2018</a></blockquote>

**Comment by jtauber** [comment]
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I think infinitive is mostly used in spanish too.<br>i.e. &quot;Abrir la puerta&quot;, &quot;Leer la nota&quot;<br>But I think a better narrative would use present in the second person form.<br>i.e. &quot;Abres la puerta&quot;, &quot;Lees la nota.&quot;<br>Can&#39;t think of many examples, as I play games in english.</p>&mdash; Alan Velasco 🐍 (@alanbato) <a href="https://twitter.com/alanbato/status/979610315066368001?ref_src=twsrc%5Etfw">March 30, 2018</a></blockquote>

**Comment by jtauber** [comment]
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">In Spanish it can be infinitive or imperative; if the latter it feels as if you are giving orders/ directions to your avatar; the former is more neutral</p>&mdash; Daniel Riaño (@danielrruf) <a href="https://twitter.com/danielrruf/status/979631581408227328?ref_src=twsrc%5Etfw">March 30, 2018</a></blockquote>

**Comment by jtauber** [comment]
I'm less interested in the intuitions of L1 English speakers that the imperative would be used (which was my initially thought although I think an infinitive might actually be what it is in English). It seems the infinitive is potentially a better candidate (at least from modern Romance language evidence).

**Comment by jtauber** [comment]
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Does it make a difference if the commands are typed or selected from a menu? Portuguese text adventures with menus use infinitives.</p>&mdash; Ken Penner (@kmpenner) <a href="https://twitter.com/kmpenner/status/980433977935843330?ref_src=twsrc%5Etfw">April 1, 2018</a></blockquote>

**Comment by seumasjeltzz** [comment]
I'm still inclined to aorist imperatives. But I'd like more data from modern European games.

**Comment by kmpenner** [comment]
Have you checked out http://docs.textadventures.co.uk/quest/translating_quest.html ?

**Comment by jtauber** [comment]
@seumasjeltzz all the feedback I got on Twitter on modern European games suggests the infinitive. I think that makes sense if you view it as completing βούλομαι... rather than commanding your avatar.

**Comment by jtauber** [comment]
@kmpenner no, but looks like a really interesting project from which we can get a lot of ideas. Although the specific page you link to doesn't directly address the question in this issue, the fact there are templates in English, French, German, Spanish, Dutch, Italian, Portuguese, Romanian, Esperanto, Russian and Icelandic means we should be able to work out what each of those languages does.

**Comment by jtauber** [comment]
Here's what the Quest language templates have for various "Take" translations:

German: Nimm
Dutch: Neem
Norwegian: Ta
French: Prendre
Italian: Prendi
Spanish: Coger
Esperanto: Coger

**Comment by jtauber** [comment]
That seems to be a mix of imperative and infinitive :-)

**Comment by seumasjeltzz** [comment]
I think I've shifted to thinking the infinitive will be fine.

**Comment by fhardison** [comment]
What about questions, either in present or future  indicative or aorist subjunctive? άρα λαμβάνεις το ...  Or something like that. Just an idea.
